server/controllers/article.js

```diff
@@ -1,12 +1,12 @@
 import models from '<serverModels>';
-import { article } from '<fixtures>/article';
 
 const { Article } = models;
 
 const createArticle = async (req, res) => {
   const { title, body } = req.body;
 
-  const categoryId = req.body.categoryId || article.categoryId;
+  const defaultCategory = process.env.DEFAULT_CATEGORY;
+  const categoryId = req.body.categoryId || defaultCategory;
```

Comment:
I thought the default category was only used for testing. Remove this from here, please. Seed the category model with categories during migrations